python/cli/prelude_cli/views/auth.py

```diff
@@ -0,0 +1,29 @@
+import click
+
+from prelude_cli.views.shared import Spinner
+
+
+@click.group()
+@click.pass_context
+def auth(ctx):
+    """Authentication"""
+    pass
+
+
+@auth.command("login")
+@click.option("-p", "--password", required=True, type=str)
+@click.pass_obj
+def login(account, password):
+    """Login using your password"""
+    with Spinner(description="Logging in and saving tokens"):
+        account.password_login(password)
+    click.secho("Login successful", fg="green")
```

Comment:
let's still use the pretty_print decorator. `results` can be an empty list and "Login successful" can go in the `message`